Corgibytes.Freshli.Cli/Functionality/Git/GitCommit.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Corgibytes.Freshli.Cli.Functionality.Git;
+
+public class GitCommit
+{
+    public readonly DateTimeOffset CommittedAt;
+    public readonly string ShaIdentifier;
+
+    public GitCommit(string shaIdentifier, DateTimeOffset committedAt)
+    {
+        ShaIdentifier = shaIdentifier;
+        CommittedAt = committedAt;
```

Comment:
Nit: A common convention (at least in C and C++, but I've seen it elsewhere) is to assign to members in the same order as they're defined. If nothing else, it improves readability, and makes it easier to spot missing member assignments.

```suggestion
        CommittedAt = committedAt;
        ShaIdentifier = shaIdentifier;
```